src/logger.go

```diff
@@ -52,3 +54,27 @@ func setupLogger(logFolder string, fileName string) *os.File {
 
 	return logFile
 }
+
+// setupSosExtrasReport sets up the sos report file for the sos_extra plugin to
+// collect the logs for the worker, which is a special file that points out to
+// the current path of the logfile for the worker.
+func setupSosExtrasReport(logFolder string, logFileName string, fileContent string) {
+	// Check if path exists, if not, create it.
+	if _, err := os.Stat(logFolder); err != nil {
+		if err := os.Mkdir(logFolder, os.ModePerm); err != nil {
+			log.Error(err)
+		}
+	}
```

Comment:
Thinking if we shouldn't make wrapper from these conditions, it's also used in setupLogger, WDYT?